HousingRegisterApi/V1/Domain/Person.cs

```diff
@@ -19,6 +19,7 @@ public Person()
         public string FirstName { get; set; }
         public string MiddleName { get; set; }
         public string Surname { get; set; }
+        public string FullName => FirstName + " " + Surname;
```

Comment:
This innocuous looking line could be the start of something significant; our API now has logic.
I think this could be a good thing. I do wonder if we'll need a strategy for this though - for example, should we now refactor all occurrences of firstname + surname away from the frontend to depend on this value instead?